style.css

```diff
@@ -391,3 +429,256 @@ textarea:hover {
+    height: auto;
+    background: linear-gradient(0deg, rgb(53, 53, 53) 0%, rgba(200, 200, 200, 0.5) 100%) url("./img/img.background.cards.png");
+    background-repeat: no-repeat;
+  }
+
+  .smoke {
+    visibility: show;
+  }
+
+  .card:hover {
+    background: url("./img/img.background.cards.png");
+    color: rgba(255, 255, 255, 0);
+    transition: 0.7s ease-in-out;
+  }
+
+  .card:hover > :nth-child(3) {
+    visibility: hidden;
+    background-color: #000;
+  }
+
+  .card .cardbottom {
+    opacity: 0;
+  }
+
+  .card:hover > .cardbottom {
+    opacity: 100;
+    transition: 0.7s ease-in-out;
+  }
+
+  .mywork {
+    width: 95vw;
+    padding: 15px;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+  }
+
+  #imageindex {
+    min-width: 600px;
+    width: 1fr;
+    height: auto;
+    padding-right: 15px;
+  }
+
+  .aboutme {
+    background-image: url("./img/img.background.aboutme.png");
+    background-position: right;
+    background-size: 100vh;
+    justify-content: center;
+    border-bottom: #fff 1px solid;
+    width: 95vw;
+    padding: 15px;
+  }
+
+  .language {
+    display: flex;
+    height: 100vh;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 93.5vw;
+    padding: 25px;
+  }
+
+  .skillstitle {
+    display: flex;
+    align-items: center;
+    padding: 20px;
+  }
+
+  .skillstitle h2 {
+    font-size: 40px;
+  }
+
+  .skills-img {
+    width: 20px;
+    height: 20px;
+    padding-bottom: 25px;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .skillitem3 li {
+    font-size: 20px;
+  }
+
+  #textaboutme {
+    max-width: 512px;
+  }
+
+  #myworktext {
+    display: flex;
+    flex-direction: column;
+    border: #000;
+  }
+
+  #worksdesktop {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  #textsubtitle {
+    margin-left: 15px;
+  }
+
+  #textparagraph {
+    margin-left: 15px;
+  }
+
+  .buttonsmywork {
+    display: flex;
+    margin: 15px;
+  }
+
+  .orange {
+    margin-left: 15px;
+    font-size: 17px;
+    border: none;
+    justify-content: center;
+    text-align: center;
+    padding: 10px 10px 10px 10px;
+  }
+
+  .containercontact {
+    margin-top: 300px;
+    display: flex;
+    flex-direction: row;
+    width: 70vw;
+  }
+
+  #formtitle {
+    padding: 0 0 0 0;
+    min-width: 49%;
+    font-size: 55px;
+    text-align: left;
+    padding-right: 15px;
+    max-height: 50vh;
+  }
+
+  #contactform {
+    padding-left: 15px;
+    max-width: 49%;
+    min-width: 49%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  #inputname {
+    margin-top: initial;
+  }
+
+  #getintouch {
+    width: 50vw;
+    font-size: 25px;
+    margin: initial;
+    padding: 15px 10px 15px 10px;
+    border: none;
+    cursor: pointer;
+    z-index: 500;
+    max-width: 200px;
+    align-items: stretch;
+    margin-top: 30px;
+  }
```

Comment:
Great project @dgonzalesi 🤝

- [ ] As specified in the project requirements, you may please implement the hover effect on all your buttons